frontend-nanodegree-arcade-game/js/app.js

```diff
@@ -117,6 +117,30 @@ const Player = class{
     }
 };
 
+ const Life = class{
+     constructor() {
+        this.sprite = 'images/Heart.png';
+        this.x = 303; this.y = randomYPosition();
+        
+     }
+     render (){
+        ctx.drawImage(Resources.get(this.sprite), this.x, this.y);
+
+     }
+     update(){
+         this.collectLife();
+
+     }
+     collectLife(){
+         console.log('error is here');
```

Comment:
should this console.log be here? Also, this code looks familiar.. isn't it already in the game?